src/Resources/app/administration/src/module/findologic-module/components/findologic-config/index.js

```diff
@@ -54,19 +59,17 @@ Component.register('findologic-config', {
   data () {
     return {
       isLoading: false,
-      term: null,
-      categories: [],
-      categoryIds: []
+      categoryCollection: null,
```

Comment:
The concept of this implementation was taken from https://github.com/shopware/platform/blob/trunk/src/Administration/Resources/app/administration/src/module/sw-sales-channel/view/sw-sales-channel-detail-base/index.js